system/Cache/Handlers/FileHandler.php

```diff
@@ -160,6 +160,30 @@ public function delete(string $key)
 
 	//--------------------------------------------------------------------
 
+	/**
+	 * Deletes items from the cache store matching a given pattern.
+	 *
+	 * @param string $pattern Cache items glob like pattern
+	 *
+	 * @return boolean
+	 */
+	public function deleteMatching(string $pattern)
+	{
+		$success = true;
+
+		foreach (glob($this->path . $pattern) as $filename)
+		{
+			if (! is_file($filename) || ! unlink($filename))
```

Comment:
Perhaps suppress the error here
```suggestion
			if (! is_file($filename) || ! @unlink($filename))
```